src/Raven.Server/Smuggler/DatabaseDataImporter.cs

```diff
@@ -82,6 +83,26 @@ public DatabaseDataImporter(DocumentDatabase database)
                                         batchPutCommand = new MergedBatchPutCommand(_database);
                                     }
                                     break;
+                                case "VersioningRevisions":
```

Comment:
I changed it to RevisionDocuments, let me know if you want to to be Revisions.
No need for what we done, as the value is always an object at this point.